calico-policies/public-network-isolation/eu-de/allow-public-services-pods.yaml

```diff
@@ -63,6 +63,11 @@ spec:
       - 158.177.129.36/32
       - 161.156.89.11/32
       - 161.156.89.12/32
+      # New IP addresses added by perik@us.ibm.com 1/19/2023
```

Comment:
In the docs here: https://cloud.ibm.com/docs/log-analysis?topic=log-analysis-endpoints  these IPs are listed as for ca-tor, so I think there might have been a copy/paste error here?